### PR TITLE
EVA-2446 - Output unmapped variants

### PR DESCRIPTION
**Pull request by tcezard**
- Output the variant that could not be mapped in a separate file
- Add retry with more memory on kill for alignment stage

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -86,14 +87,14 @@ include { process_split_reads; process_split_reads_mid; process_split_reads_long
 /*
  * Create the header for the output VCF
  */
-process buildHeader {
+process convertAndAddHeaderToVCF {
```
Comment (andresfsilva) [inline main.nf:90]: what if we change the process name to something like `generateRemappedVCF` and add to the description that we are coverting to VCF and merging the header with the remapped variants?

File: main.nf [hunk 2]
```diff
@@ -113,24 +114,29 @@ process buildHeader {
...
+    cat final_header.txt variants_remapped_sorted.vcf > variants_remapped_sorted_with_header.vcf
     """
 }
 
 /*
- * Add header to output VCF and merge with reference allele file
+ * Add header to unmapped variant VCF records
  */
-process mergeHeaderAndContent {
+process mergeOriginalHeaderAndVCFAndOutput {
```
Comment (andresfsilva) [inline main.nf:124]: I think having the details in the comment description and a more general name helps to read the pipeline easier. Like renaming the process to something like `generateUnmappedVCF` and then explaining in the description that we are doing the merging

**Review by andresfsilva — APPROVED** [review]

**Review by apriltuesday — APPROVED** [review]

File: main.nf (hunk 1 above)
Comment (tcezard) [reply main.nf:90]: I've address this comment in subsequent PR [EVA-2451 - allow novel ref allele #29
](https://github.com/EBIvariation/variant-remapping/pull/29)

File: main.nf (hunk 2 above)
Comment (tcezard) [reply main.nf:124]: I've address this comment in subsequent PR [EVA-2451 - allow novel ref allele #29
](https://github.com/EBIvariation/variant-remapping/pull/29)

**Pull request merged**